kodilla-testing/src/main/java/com/kodilla/testing/forum/ForumUser.java

```diff
@@ -17,41 +17,55 @@ public ForumUser(String name, String realName) {
     }
 
     public void addPost(String author, String postBody){
-        // do nothing
+        ForumPost thePost = new ForumPost(postBody, author);
+        posts.add(thePost);
     }
 
     public void addComment(ForumPost thePost, String author, String commentBody){
-        // do nothing
+       ForumComment theComment = new ForumComment(thePost, commentBody, author);
+       comments.add(theComment);
     }
 
     public int getPostsQuantity(){
-        // return 100 temporarily
-        return 100;
+        return posts.size();
     }
 
     public int getCommentsQuantity(){
-        // return 100 temporarily
-        return 100;
+        return comments.size();
     }
 
     public ForumPost getPost(int postNumber){
-        // returning null means that the operation was unsuccessful
-        return null;
+        ForumPost thePost = null;
+        if (postNumber >= 0 && postNumber < posts.size()) {
+            thePost = posts.get(postNumber);
+        }
+        return thePost;
     }
 
     public ForumComment getComment(int commentNumber){
-        // returning null means that the operation was unsuccessful
-        return null;
+        ForumComment theComment = null;
+        if (commentNumber >= 0 && commentNumber < comments.size()) {
+            theComment = comments.get(commentNumber);
+        }
+        return theComment;
     }
 
     public boolean removePost(ForumPost thePost){
-        // return true temporarily
-        return true;
+        boolean result = false; //Can be done better using one line - reading remove docs - return posts.remove(thePost);
```

Comment:
Uwaga jak w Twoim komentarzu :)

Reply:
Tak. Tylko ze to była wersja z kodilla wiec nie chciałem tego zmieniać. Napisałem komentarz dla siebie w przyszłości.